core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,25 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, inlcude_deleted=False):
```

Comment:
Add another argument for `Cursor`. When it is None return first 10 jobs but when it is not None fetch 10 job starting from that Cursor.